messages/commonFlags.md

```diff
@@ -0,0 +1,53 @@
+# promote.devops-center-project-name.summary
+
+Name of the DevOps Center project.
+
+# promote.branch-name.summary
+
+Name of the branch from which to deploy changes to the stage’s org.
```

Comment:
This is slightly different from the docs: "Name of the branch in the source control repository from which to deploy changes to the stage’s org."

Reply:
Fixed, thank you.